packages/run-protocol/src/metrics.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+import { makeStoredSubscription, makeSubscriptionKit } from '@agoric/notifier';
+import { E } from '@endo/eventual-send';
+
+const PATH = 'metrics';
+
+/**
+ * @template T
+ * @param {ERef<StorageNode>} [storageNode]
+ * @param {ERef<Marshaller>} [marshaller]
+ * @param {T[]} optionalInitialState
+ * @returns {MetricsSubscriptionKit<T>}
+ */
+export const makeMetricsSubscriptionKit = (
```

Comment:
If we want to make it so that we never have to change the contracts using the helper again, I'd add the following to `packages/notifier/src/storesub.js`:

```js
/**
 * @template T
 * @typedef {object} StoredPublisherKit
 * @property {StoredSubscription<T>} subscriber TODO: change to StoredSubscriber<T> when available
 * @property {IterationObserver<T>} publisher
 */

/**
 * @template T
 * @param {T} initialState
 * @param {ERef<StorageNode>} [storageNode]
 * @param {ERef<Marshaller>} [marshaller]
 * @param {string} [childPath]
 * @returns {StoredPublisherKit<T>}
 */
export const makeStoredPublisherKit ...
```

then import it as follows:

```js
import { makeStoredPublisherKit } from '@agoric/notifier';
```